src/styles/GoalPage.module.css

```diff
@@ -0,0 +1,53 @@
+.goalDescriptionContainer {
+  background-color: #ffffff;
```

Comment:
You can use the global-defined variable for colors instead of the color palette. All the global variables are defined in globals.css

Reply:
Got it. There was one color I kept as is since there was no global variable to handle it, but the others I found and utilized.